docs/en/docs/tutorial/extra-data-types.md

```diff
@@ -55,108 +55,4 @@ Here are some of the additional data types you can use:
 
 Here's an example *path operation* with parameters using some of the above types.
 
-//// tab | Python 3.10+
-
-```Python hl_lines="1  3  12-16"
-{!> ../../docs_src/extra_data_types/tutorial001_an_py310.py!}
-```
-
-////
-
-//// tab | Python 3.9+
-
-```Python hl_lines="1  3  12-16"
-{!> ../../docs_src/extra_data_types/tutorial001_an_py39.py!}
-```
-
-////
-
-//// tab | Python 3.8+
-
-```Python hl_lines="1  3  13-17"
-{!> ../../docs_src/extra_data_types/tutorial001_an.py!}
-```
-
-////
-
-//// tab | Python 3.10+ non-Annotated
-
-/// tip
-
-Prefer to use the `Annotated` version if possible.
-
-///
-
-```Python hl_lines="1  2  11-15"
-{!> ../../docs_src/extra_data_types/tutorial001_py310.py!}
-```
-
-////
-
-//// tab | Python 3.8+ non-Annotated
-
-/// tip
-
-Prefer to use the `Annotated` version if possible.
-
-///
-
-```Python hl_lines="1  2  12-16"
-{!> ../../docs_src/extra_data_types/tutorial001.py!}
-```
-
-////
-
-Note that the parameters inside the function have their natural data type, and you can, for example, perform normal date manipulations, like:
```

Comment:
This text should be kept, the include below is a different one from the previous one.

Reply:
@tiangolo thank you for the feedback. I have updated it!